### PR TITLE
Ui: Add contributing pattern doc

### PR DESCRIPTION
**Pull request by hellobontempo**

### CODE REVIEW

File: ui/docs/css.md
Comment (Monkeychip) [inline ui/docs/css.md:1]: Thank you for putting this together. I'll fill this out post PR work. There are still some patterns that are working themselves out and I'll have a better sense of them post project (ex: sizing vars).

File: ui/docs/models.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  get canRead() {
+    return this.fooPath.get('canRead') !== false;
+  }
+  get canEdit() {
+    return this.fooPath.get('canUpdate') !== false;
+  }
+}
+```
+
+## Decorators
```
Comment (hashishaw) [inline ui/docs/models.md:40]: I should add some content about `withExpandedAttributes` and why you might use that one or `withFormFields`. 
Happy to add that to this PR, or as a follow-on

**Review by hashishaw — APPROVED** [review]
Great teamwork putting this together! Looking forward to seeing it evolve over time

File: ui/docs/models.md (hunk 1 above)
Comment (hellobontempo) [reply ui/docs/models.md:40]: Whichever is easiest for you! I was planning to wait to merge next week until all of the release branch work is wrapped up. Feel free to add that to this PR. 

I agree, that content would be super helpful 💡

**Pull request merged**